core/src/apps/nem/sign_tx.py

```diff
@@ -25,6 +28,7 @@ async def sign_tx(ctx, msg: NEMSignTx, keychain):
     node = keychain.derive(msg.transaction.address_n)
 
     if msg.multisig:
+        assert msg.multisig.signer is not None
```

Comment:
Done in 329a038 - I am checking it and raising if None. It probably cannot be made required in protobuf, as this `NEMTransactionCommon` message type is not used only on the `multisig` property, but also on `transaction`, and it can be empty there (I did not see otherwise)